dockerfiles/compilation/java-8/integration_tests.sh

```diff
@@ -53,5 +53,5 @@ git clone $ORIGIN/.
 git checkout $SHA1
 
 
-mvn -DskipTests -pl org.apache.james:apache-james-mpt-external-james -am package
+mvn -DskipTests -pl org.apache.james:apache-james-mpt-external-james -am install
```

Comment:
this will break the cache for future build, no?

Reply:
of course, it requires an update of the jenkins side